src/update_flash.c

```diff
@@ -148,6 +150,74 @@ static int wolfBoot_copy_sector(struct wolfBoot_image *src, struct wolfBoot_imag
     return pos;
 }
 
+#ifdef DELTA_UPDATES
+static uint8_t delta_blk[WOLFBOOT_SECTOR_SIZE];
```

Comment:
On the STM32F4 the sector size is 128KB right? Is there a better way to handle this? Should this be "DELTA_BLOCK_SIZE"?

Reply:
Good point. I've reworked the code to use a fixed size for applying the patch during updates. Patch can be applied/written 256B at a time now, regardless of the physical sector size.